<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientWidget</class>
 <widget class="QWidget" name="clientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leipAddress">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border:2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leport">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border:2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius:10px;
border:2px solid #000000;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #FF3030;
}</string>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border:2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
background-color:rgba(0,0,0,0);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textSend">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid #000000;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBtn_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-radius:10px;
border:2px solid #000000;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #FF3030;
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textReceive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid #000000;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
